src/boot/efi/disk.c

```diff
@@ -32,7 +32,27 @@ EFI_STATUS disk_get_part_uuid(EFI_HANDLE *handle, char16_t uuid[static 37]) {
                 if (hd.SignatureType != SIGNATURE_TYPE_GUID)
                         continue;
 
-                GuidToString(uuid, (EFI_GUID *) &hd.Signature);
+                _cleanup_free_ char16_t *tmp = xasprintf(
```

Comment:
OOM check?

Reply:
Our efi code panics on allocation failures, hence the functions that start with `x`.